### PR TITLE
Add github actions workflow and remove travis ci config

### PR DESCRIPTION
**Pull request by jregistr**
*Issue #, if available:*

N/A

*Description of changes:*
Similar to some other providers https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds, Update Kendra Provider to use github actions for CI actions on pull request and push to master for verification.

Also ran the pre-commit script.

-----

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jregistr** [comment]
To test this locally, use https://github.com/nektos/act, preferably on a Linux system.

File: .github/workflows/maven-verify.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        run: mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean install --no-transfer-progress
+
+
+      - name: Verify AWS::Kendra::Faq
+        working-directory: aws-kendra-faq
+        run: mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean verify --no-transfer-progress
+
+      - name: Install AWS::Kendra::Faq
+        working-directory: aws-kendra-faq
+        run: mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean install --no-transfer-progress
```
Comment (kevinawskendra) [inline .github/workflows/maven-verify.yml:59]: Do we need to do the same for featured result set?

Comment (jregistr) [reply .github/workflows/maven-verify.yml:59]: Yup, I had missed this one. Updated config. To keep the structure of the commits, I edited the first commit to add FRS and did a force push to the branch.

**Review by kevinawskendra — APPROVED** [review]

File: aws-kendra-datasource/aws-kendra-datasource.json [hunk 2]
```diff
@@ -1201,7 +1201,7 @@
     },
     "WebCrawlerSeedUrl": {
       "type": "string",
-      "pattern": "^(https?):\/\/([^\\s]*)",
+      "pattern": "^(https?)://([^\\s]*)",
```
Comment (tzamazon) [inline aws-kendra-datasource/aws-kendra-datasource.json:1204]: Could this change lead to backward incompatible?

Comment (jregistr) [reply aws-kendra-datasource/aws-kendra-datasource.json:1204]: These are the same things. The `/` is not a special character, so you don't need to escape it.

Comment (jregistr) [reply aws-kendra-datasource/aws-kendra-datasource.json]: Alright, to be safe, removed this lint rule and reverted the change.

**Review by tzamazon — APPROVED** [review]

**Review by kevinawskendra — APPROVED** [review]

**Pull request merged**